metadata-patch.md

```diff
@@ -0,0 +1,195 @@
+# Modifying metadata via Patch
+[Back to the index](README.md)
+
+**Contents:**
+
+* [Introduction](#introduction)
+* [Adding metadata](#adding-metadata)
+* [Removing metadata](#removing-metadata)
+* [Replacing metadata](#replacing-metadata)
+* [Ordering metadata](#ordering-metadata)
+
+## Introduction
+
+A user with write permission on a DSpace object may change any of its metadata
+using an HTTP PATCH request. All DSpace object types (Bitstreams, Items, Collections, etc.)
+support metadata changes in roughly the same way.
+
+This guide is focused on metadata-only changes and omits non-metadata parts of the object's
+json representation for brevity. However, you should be aware that PATCH requests operate
+against the whole object, and may therefore modify other mutable attributes in the same
+request.
+
+For additional information about PATCH support in DSpace, including request format and response
+status code details, see [General rules for the Patch operation](patch.md)
+
+The examples in each section below build on each other, assuming an initial metadata state of:
+
+```json
+{
+  "metadata": {
+    "dc.title": [
+      { "value": "Initial Title", "language": null, "authority": null, "confidence": -1 }
+    ]
+  }
+}
+```
+
+## Adding metadata
+
+With the `add` operation, you can add any number of metadata values in a single request.
+
+Note:
+
+* When no values exist yet for a metadata key, you must add the key and value array with one
+  or more values.
+* When values already exist for a metadata key, it is necessary to specify the new value's
+  position at the end of the `path` (`/0` means first position, `/-` means last).
+* When creating new metadata values, if unspecified, the `language` and `authority` properties
+  default to `null`, and `confidence` defaults to `-1`.
+
```

Comment:
It should be stated that the add operation can be used to replace the metadata, see analog comment here (the previous paragraph)
https://github.com/DSpace/Rest7Contract/blob/master/workspaceitem-data-metadata.md#remove

> This use of the add operation to replace the list of values of a metadata could be counter intuitive but it is done according to the RFC section 4.1 ...

Reply:
I agree, this is another area of possible borrowing/consolidation.  At a minimum, we should align these two documentation sections.

Reply:
done in latest commit